backend/app/services/adapters/task_kinds/queries.py

```
@@ -315,8 +810,275 @@ def get_task_detail(
+                "attachments": [],
+                "sender_type": subtask.sender_type,
+                "sender_user_id": subtask.sender_user_id,
+                "sender_user_name": getattr(subtask, "sender_user_name", None),
+                "reply_to_subtask_id": subtask.reply_to_subtask_id,
+            }
+            subtasks_dict.append(subtask_dict)
+
+        return subtasks_dict
+
+    def _add_group_chat_info_to_task(
+        self, db: Session, task_id: int, task_dict: Dict[str, Any], user_id: int
```

Ruff (0.15.5)

[warning] Unused method argument: `user_id`

(ARG002)